controllers/cloud.redhat.com/providers/kafka/localkafka.go

```diff
@@ -43,6 +46,12 @@ func (k *localKafka) CreateTopics(app *crd.ClowdApp) error {
 				RequestedName: topic.TopicName,
 			},
 		)
+
+		conn, err := kafka.DialLeader(k.Ctx, "tcp", host, topicName, 0)
```

Comment:
I am guessing this results in a single partition on the topic, right? Do we need to go with the second approach here so we can create the topic with the explicit number of partitions specified in our `ClowdEnvironment KafkaTopicSpec`?

https://github.com/segmentio/kafka-go#to-create-topics